src/Hyperion.Benchmarks/SerializeClassesBenchmark.cs

```diff
@@ -20,9 +21,25 @@ public class SerializeClassesBenchmark : HyperionBenchmark
         private LargeSealedClass sealedObject;
         private GenericClass<int, string, bool, DateTime, Guid> genericObject;
 
+        private Serializer _filteredSerializer;
+        
         protected override void Init()
         {
-            Serializer = new Serializer(new SerializerOptions(preserveObjectReferences:true));
+            var baseOptions = new SerializerOptions(preserveObjectReferences: true);
+            Serializer = new Serializer(baseOptions);
```

Comment:
Is this serving a different purpose compared to `_filteredSerializer`? if not, then we can do this `Serializer = _filteredSerializer;`.

Reply:
Yeah that's an interesting question - why do we have two different serializer instances here for these benchmarks?

Reply:
`Serializer` is the one that came with the benchmark spec, its the one with the type filtering disabled, we're benchmarking 2 different serializer, one disabled and one with type filtering turned on.